frontend-react/src/hooks/UseAdminSafeOrganizationName.test.ts

```diff
@@ -0,0 +1,72 @@
+import { renderHook } from "@testing-library/react-hooks";
+
+import { orgServer } from "../__mocks__/OrganizationMockServer";
+import { mockSessionContext } from "../contexts/__mocks__/SessionContext";
+import { useSessionContext } from "../contexts/SessionContext";
+
+import { MemberType } from "./UseOktaMemberships";
+import {
+    Organizations,
+    useAdminSafeOrganizationName,
+} from "./UseAdminSafeOrganizationName";
+
+describe("useAdminSafeOrganizationName", () => {
+    beforeAll(() => {
+        orgServer.listen();
+    });
+    afterEach(() => orgServer.resetHandlers());
+    afterAll(() => orgServer.close());
+    test("returns undefined if no active membership parsed name", () => {
+        mockSessionContext.mockReturnValue({
+            oktaToken: {
+                accessToken: "TOKEN",
+            },
+            activeMembership: undefined,
+            dispatch: () => {},
+            initialized: true,
+        });
+        const { activeMembership } = useSessionContext();
+        const { result } = renderHook(() =>
+            useAdminSafeOrganizationName(activeMembership?.parsedName)
+        );
+        expect(result.current).toEqual(undefined);
+    });
+    test("returns correct client organization", async () => {
```

Comment:
Minor point, but I don't think this test and [the next one](https://github.com/CDCgov/prime-reportstream/pull/7521/files#diff-9c94215ec20e8aa9cff0f56f854318feb86d24eabd23c6deab16ca9ec693da1dR53) need to be `async` since they're not `await`ing anything.